client/components/App.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+	<div
+		id="viewport"
+		role="tablist">
+		<aside id="sidebar">
+			<div class="scrollable-area">
+				<div class="logo-container">
+					<img
+						:src="'img/logo-' + ( isPublic() ? 'horizontal-' : '' ) + 'transparent-bg.svg'"
```

Comment:
Nit: I believe our JS files are set up to have no spaces inside parentheses, not sure. I think that would make sense.

Also thoughts on:

```suggestion
						:src="`img/logo-${isPublic() ? 'horizontal-' : ''}transparent-bg.svg`"
```

(None of this is holding off the PR of course, just thinking out loud)

Reply:
Hm, I'll look around maybe there's a way to fix eslint to apply inside of vue strings like that.